src/lib.rs

```diff
@@ -67,9 +67,9 @@ pub mod value;
 
 pub mod extensions;
 
-pub use de::{from_str, Deserializer};
+pub use de::{from_bytes, from_reader, from_str, Deserializer};
```

Comment:
if we are re-exporting everything, let's make `ser` and `de` private

Reply:
This, however, will be a breaking change and would require a major version bump.

Reply:
I'd want to get it anyway